components/pets/table.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react'
+import axios from 'axios'
+import { Pet } from '../../models/pet';
+import { baseUrl } from '@/baseUrl'
+import PetsRow from './tableRow'
+import { pets as seedPets } from '@/seeds';
+
+const PetsTable:React.FC = () => {
+  // const [pets, setPets] = useState<Pets[]>(seedPets)
+  const [pets, setPets] = useState<Pet[]>([])
+  
+  const loadPets = () => {
+    axios
+      .get(`${baseUrl}pets`)
+      .then((res) => {
+        console.log(res.data)
+        setPets(res.data);
+      })
+      .catch((error) => console.log(error));
+  }
+
+  useEffect(() => {
+    loadPets()
+  }, [pets])
+  
+  return(
+    <table>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Species</th>
+          <th>Gender</th>
+          <th>Age</th>
+        </tr>
+      </thead>
+      {pets ? 
```

Comment:
I think this ternary can be `pets.length > 0` because pets defaults to an empty array, meaning the false version of this will never get hit. Just an empty table.